src/backend/Main.mo

```diff
@@ -286,6 +286,9 @@ shared (install) actor class DAO(localDeploymentCanisterIds : ?{ btcflower : Tex
         if (args.option >= proposal.options.size()) {
           return #err("Proposal " # debug_show (args.proposalId) # " does not have an option " # debug_show (args.option));
         };
+
+        let stakedVotingPower = await getUserStakedVotingPower(caller);
```

Comment:
i think for the `stakedVotingPower` we also have to return the mint numbers of the flowers, otherwise someone could vote with his staked flowers multiple times. see the following example:
- stake on btc flower and hold one btc flower in wallet
- vote on proposal, voting power is 4 (2 from staked and 2 from wallet)
- send flower to new wallet
- buy another btcflower on initial wallet and vote again, voting power is 4 again (2 from staked and 2 from wallet)